cg/cli/downsample.py

```diff
@@ -0,0 +1,74 @@
+"""cg module for downsampling reads in a sample."""
+
+import logging
+from pathlib import Path
+from typing import Tuple
+
+import click
+
+from cg.apps.downsample.downsample import DownsampleAPI
+from cg.apps.downsample.utils import store_downsampled_sample_bundle
+from cg.constants.constants import DRY_RUN
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def downsample():
+    """Downsample reads in a sample."""
+
+
+@downsample.command("samples", help="Downsample reads in one or multiple samples in a case.")
```

Comment:
In addition to the comment below - I think that you can give an elaborate full use case with the help of this command - one being for one sample to downsample, and also for multiple ones to downsample. Alternatively, just write that you can only supply one sample and give an example of a multiple sample case instead

Reply:
I added a use case that also indicates that one can supply values below 1